Lab 2.ipynb

```diff
@@ -312,19 +312,38 @@
     "final_3 = final_3 / np.sum(final_3)\n",
     "k3 = np.linspace(0, 20, len(final_3))\n",
     "\n",
-    "fig, ax = plt.subplots(1, 1, figsize=(8,8))\n",
+    "fig, ax = plt.subplots(1, 2, figsize=(15,8))\n",
     "\n",
     "plt.tick_params(labelsize = 12)\n",
-    "ax.set_xlim([0,10])\n",
-    "ax.set_ylabel('Probability', fontsize = 12)\n",
-    "ax.set_xlabel('Value')\n",
-    "\n",
-    "ax.plot(k1, final_1)\n",
-    "ax.plot(k2, final_2)\n",
-    "ax.plot(k3, final_3)\n",
-    "ax.set_xlim([0,4])\n",
-    "\n",
-    "ax.legend([\"Day \" + str(N1), \"Day \" + str(N2), \"Day \" + str(N3)])"
+    "ax[0].set_title(\"Rayleigh Distributions Averaged\")\n",
+    "ax[0].set_ylabel('Probability', fontsize = 12)\n",
+    "ax[0].set_xlabel('Value')\n",
+    "ax[1].set_title(\"Rayleigh Distributions Averaged (Log Plot)\")\n",
+    "ax[1].set_ylabel('Probability (log)', fontsize = 12)\n",
+    "ax[1].set_xlabel('Value')\n",
+    "\n",
+    "ax[0].plot(k, original / np.sum(original))\n",
+    "ax[0].plot(k1, final_1)\n",
```

Comment:
I like that you did unevenly spaced distributions and went all the way to 50, it shows it becoming Gaussian well without crowding the graph